ops/instrument_httpclient.go

```diff
@@ -0,0 +1,145 @@
+package ops
+
+import (
+	"context"
+	"time"
+
+	"github.com/gofiber/fiber/v2/utils"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/valyala/fasthttp"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/metric/unit"
+	"go.opentelemetry.io/otel/semconv/v1.17.0"
+
+	"github.com/valkyrie-fnd/valkyrie/internal"
+	"github.com/valkyrie-fnd/valkyrie/rest"
+)
+
+func init() {
+	rest.Pipeline.Register(httpTracingHandler(), httpLoggingHandler(), httpMetricHandler())
```

Comment:
A bit of the same comment as for the other one. The registration feels a bit hidden in an `init` like this.

Reply:
Originally I had a public function `ConfigureInstrumentation()` called by the server package, perhaps thats more clear?

Reply:
I think the order is what's a bit confusing for me. It seems the middleware/pipelineHandlers "decide" where they should be added, instead of where it is used "deciding" what middleware/pipelines to use.

I think that would be a bit more with a public function. 
Otherwise you have and use a `Pipeline` in `rest/genericpam/vplugin` but unclear what handlers are actually added to them. 

But if the order that is now is needed in order to keep a better package hierarchy, maybe it's all right?

Reply:
Yeah I understand that its a bit confusing and can clarify with comments that the Pipeline structs are intentionally there in rest/genericpam/vplugin to allow for custom handlers to be registered in these critical sections (such as logging, tracing etc) from outside

Reply:
And I can look into changing registering them in `init()` in ops to something more configurable